src/transformers/modeling_utils.py

```diff
@@ -5013,6 +5014,41 @@ def _is_quantized_training_enabled(self):
 
         return self.hf_quantizer.is_trainable
 
+    def tensor_parallel(self, device_mesh):
```

Comment:
@kwen2501 Is there a way to know if tensor_parallel took effect or not? For instances like there was no usable tp plan found in any sub module so no TP took place etc. Returning that indication would help in better downstream usage. 

cc: @ArthurZucker

Reply:
I added logging in this method. You can see if by setting `TRANSFORMERS_VERBOSITY=DEBUG`.

Reply:
Logging is great, but need some indication that can be consumed programatically :)